src/main/resources/config/application.yaml

```diff
@@ -23,12 +23,20 @@ spring:
           destination: ${powsybl-ws.rabbitmq.destination.prefix:}loadflow.cancel
         publishStopped-out-0:
           destination: ${powsybl-ws.rabbitmq.destination.prefix:}loadflow.stopped
-        publishFailed-out-0:
-          destination: ${powsybl-ws.rabbitmq.destination.prefix:}loadflow.failed
         publishCancelFailed-out-0:
           destination: ${powsybl-ws.rabbitmq.destination.prefix:}loadflow.cancelfailed
-      output-bindings: publishRun-out-0;publishResult-out-0;publishCancel-out-0;publishStopped-out-0;publishFailed-out-0;publishCancelFailed-out-0
-
+      output-bindings: publishRun-out-0;publishResult-out-0;publishCancel-out-0;publishStopped-out-0;publishCancelFailed-out-0
+      rabbit:
+        bindings:
+          consumeRun-in-0:
+            consumer:
+              auto-bind-dlq: true
+              dead-letter-exchange: ${powsybl-ws.rabbitmq.destination.prefix:}loadflow.run.dlx
+              dead-letter-queue-name: ${powsybl-ws.rabbitmq.destination.prefix:}loadflow.run.dlx.dlq
+              dead-letter-exchange-type: topic
+              quorum:
+                enabled: true
+                delivery-limit: 2
```

Comment:
I'm not sure, but in my deployment I needed to remove manually the existing queue (that was a classic queue). What to do on our deployments?

Reply:
in dev since we don't persist data, docker compose down and then up is enough to force to recreate the queues. You don't need a special branch in the deployment, explicit quorum queues are already supporter in the rabbitmq version that we use on main (3.13.x)

Reply:
I was thinking "what to do for our dev/rec/production envs?" as I did manual actions on my local deployment